File: pre-contest/a_bus_ride/sol/a_bus_ride.java
import java.util.Scanner;

public class MainShaker {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System. in);
        int nbPerson = Integer.parseInt(scanner.nextLine());
        int nbSeat = Integer.parseInt(scanner.nextLine());
        
        System.out.println((int)Math.ceil(nbPerson / (double)nbSeat));
        
        scanner.close();
    }
}

File: pre-contest/a_bus_ride/sol/a_bus_ride.cs
using System; 
  
class GFG { 
  
    // Main Method 
    static public void Main(String[] args) 
    {
		int nbPerson = int.Parse(Console.ReadLine());
		int nbSeat = int.Parse(Console.ReadLine());

		Console.WriteLine(Math.Ceiling(nbPerson / (double) nbSeat));
    } 
} 
